.github/workflows/compliance.yml

```diff
@@ -1,15 +1,20 @@
 name: Compliance Checks
 
 on:
-  pull_request:
-    types:
-    - edited
```

Comment:
I think this is going to break the "empty PR message" below, the check would fail and only rerun on push. It should probably be moved on a separate workflow.

Reply:
yeah i know and I am aware of that. I am thinking of moving this type of checks to somewhere else.

Reply:
@fabiobaltieri  should be "fixed" now